src/lib/command_line.ml

```diff
@@ -71,6 +92,40 @@ let root_term () =
     ~doc:"The root of the configuration"
 
 
+let client_term =
+  let open Cmdliner in
+  let term =
+    let open Term in
+    pure (fun (`Base_url base_url) action ids ->
+        let res = client ~base_url action ids in
+        match Lwt_main.run res with
+        | `Error e ->
+          eprintf "An error has occured.\n";
+          exit 2
+        | `Ok () -> ()
+      )
```

Comment:
There is a `run_deferred` function that display the errors.
https://github.com/hammerlab/coclobas/blob/master/src/lib/command_line.ml#L54